src/components/ble/AlertNotificationService.h

```diff
@@ -24,8 +27,25 @@ namespace Pinetime {
         int OnAlert(uint16_t conn_handle, uint16_t attr_handle,
                                     struct ble_gatt_access_ctxt *ctxt);
 
+        void event(char event);
+
+        static const char EVENT_HANG_UP_CALL = 0x00;
+        static const char EVENT_ANSWER_CALL = 0x01;
```

Comment:
From my experience with huami, perhaps the options should be hangup and ignore, not answer.  If you need to answer the call, you need the phone anyway.

Reply:
If you are wearing headphones it can be useful but not sure how often it is necessary. I can see myself out on a run/bicycle and using headphones for music and wanting to accept the call with the watch

Reply:
Yeah, I cant disagree